### PR TITLE
add more examples to readme

### PR DESCRIPTION
**Pull request by khimaros**
clarify use of client ids and auth tokens as well as pointing to an alternative sync server.

### CODE REVIEW

**Comment by clach04** [comment]
Thanks for posting this @khimaros, this definitely helps get me closer!

Initially the the payloads(e.g. bookmarks) are not human readable for me but that was due to me missing the `--decrypt` flag on the command line (I'd been using my old fork that auto-decrypted and forgot about the flag).

Thanks for posting.


@enote-kane as you appear to be the last person with commit permissions to make a commit please could this PR from @khimaros be merged? Thanks!

File: README.rst [hunk 1]
```diff
@@ -7,15 +7,31 @@ This is a python client for Firefox Sync. Check it out with::
...
+  $ python syncclient/main.py -u "me@example.com" --client-id <YOUR CLIENT ID> get_records password
   [u'{1c1e0eea-d9c2-4c59-b95e-4dbe0800639f}',
    u'{0a76ec08-ba7c-48b1-b026-1d65085f789e}',
    u'{7482b391-bf2f-4542-8ebd-27c4398487ff}',
    u'{37bc9298-ac49-c54e-a73d-d817434ed0b2}',
    u'{d5ff4718-d4a0-4703-b0af-7d1c79c3a099}']
 
+If you want to get all bookmarks (decrypted)::
+
+  $ python syncclient/main.py -u "me@example.com" --client-id <YOUR CLIENT ID> get_records bookmarks
```
Comment (clach04) [inline README.rst:33]: needs ` --decrypt` flag (likeline 37) to match claim on line 31.

**Review by ghost — APPROVED** [review]
LGTM, thanx for contributing

File: README.rst (hunk 1 above)
Comment (ghost) [reply README.rst:33]: I'll address that separately.

**Pull request merged**